scalableminds/webknossos
# Allow to share annotations for non-public datasets

Make it possible to share annotations for non-public datasets. The dataset sharing token is only appended to the link if the dataset is not public. Also users need to copy the sharing link from the share modal, as the normal URL doesn't contain the dataset sharing token, but I'd say that's fine, wdyt?

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create a tracing in a non-public dataset. Make the tracing public using the "Share" functionality and copy the link in the Share modal. Log out and paste the copied link into the address bar. You should be able to access the tracing.

### Issues:
- fixes #3599 

------
